woocommerce/woocommerce-android
# Display manuals per country

Closes: #7194 


### Description


This PR:

-  removes the Card Reader Manuals row from the IPP Hub when the store is not in the US or Canada, and of course, shows the row only if it's US and Canada. 
- Adds the logic so, when on a Canadian store, only the wisepad3 Manual is displayed and when on a US store only the M2 and BBPOS chipper are displayed

### Testing instructions


Please test using a store in the US, a store in Canada and a store o an unsupported country:

US Store:
- Go to Menu - Payments -> Make sure the row `Card Reader Manuals` is displayed
- Click on `Card Reader Manuals` -> Make sure both the BBPOS Chipper and the M2 reader are displayed (and only those)

Canadian Store:
- Go to Menu - Payments -> Make sure the row `Card Reader Manuals` is displayed
- Click on `Card Reader Manuals` -> Make sure  the Wisepad3 is displayed (and only this one)

Unsupported Store:
- Go to Menu - Payments -> Make sure the row `Card Reader Manuals` is **NOT** displayed


### Images/gif

![manualsScreen](https://user-images.githubusercontent.com/30724184/188035232-a51a8778-5759-44f7-b333-492a6f1a4239.gif)
